lambdas/initialise-ipv-session/src/main/java/uk/gov/di/ipv/core/initialiseipvsession/InitialiseIpvSessionHandler.java

```diff
@@ -174,8 +174,6 @@ public APIGatewayProxyResponseEvent handleRequest(
                             auditEventUser,
                             reproveAuditExtension);
 
-            LOGGER.warn("Audit Event: " + auditEvent.getExtensions());
-
```

Comment:
Is this deliberate?

Reply:
I accidentally left this in on the previous PR that got merged in, was just using it to debug and feels unnecessary given we can access the audit events from the auditing lambda anyway

Reply:
We don't ever want to log audit events/extensions in prod given they may contain PII - I don't think it's an issue in this case as it will only have been logging null in prod here but one to be careful of